internal/providers/packet/packet.go

```diff
@@ -48,3 +62,102 @@ func FetchConfig(f resource.Fetcher) (types.Config, report.Report, error) {
 
 	return util.ParseConfig(f.Logger, data)
 }
+
+// FetchStatus is invoked by oem.Status() when igintion fails
+// POSTS if s==fail, a message to Packet's provision setup Timeline
+func FetchStatus(f resource.Fetcher, s string) error {
+
+	type Message struct {
+		Message string `json:"message"`
+	}
+
+	// fetch JSON from https://metadata.packet.net/metadata
+	data, err := f.FetchToBuffer(MetadataUrl, resource.FetchOptions{
+		Headers: nil,
+	})
+	if data == nil {
+		log.Fatalf("data is nil")
```

Comment:
return err here instead of just logging, unless theres a reason not to. In general do that throughout this file